doc/man7/openssl_user_macros.pod.in

```diff
@@ -23,21 +23,16 @@ user defined macros.
 
 =item B<OPENSSL_API_COMPAT>
 
-The value is a version number similar to the
-L<OPENSSL_VERSION_NUMBER(3)> macro.  Any symbol that is deprecated in
-versions up to and including the version given in this macro will not
-be declared.
-
-The version number assigned to this macro can take one of two forms:
+The value is a version number, given in one of the following two forms:
 
 =over 4
 
 =item C<0xMNNFF000L>
 
 This is the form supported for all versions up 1.1.x, where C<M>
 represents the major number, C<NN> represents the minor number, and
-C<FF> represents the fix number.  For version 1.1.0, that's
-C<0x10100000L>.
+C<FF> represents the fix number, as a hexadecimal numbers.
```

Comment:
Should this be `as hexadecimal numbers`?

Reply:
Ah, no, "numbers" should be "number". Good catch, though!

Reply:
Fixed